docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -47,6 +57,22 @@ If there is an error please [contact support](mailto:pulsesupport@codacy.com).
 -   The set of changes in a deployment is the list of commits in the corresponding pull request.
 -   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
 
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The following is a detailed description of how the Pulse Bitbucket integration automatically detects incidents:
+
+### Use pull request reverts (based on default branch)
+
+-   Pulse considers an incident a reverted pull request that **targets the default branch** of the repository when the revert operation was executed through the Bitbucket UI originating a branch which name starts with `revert-`.
+-   The incident creation date is the timestamp when the reverted pull request was initially merged.
+-   Pulse associates incidents to the system matching the repository name.
```

Comment:
We also associate teams in the same way as other PRs. We look at the teams of the commit authors.

It is possible that a PR was created by team A but is reverted by team B. So the incident will be associated with team B. /cc @mrfyda

Reply:
not sure if I understood this example ☝️ 

from what I understand:
1. pull request merged by team A => Deployment 1 for team A
2. pull request revert merged by team B => Deployment 2 for team B, Incident 1

given that Incidents don't have teams, and are filtered by their associated Deployments, in this case, Incident 1 would be associated with Deployment 1, therefore being accounted for team A, no?

Reply:
Yes, you are absolutely right 🤦 . I completely forgot that incidents don't have teams.